### ISSUE TITLE
论文中的K是什么意思

### ISSUE THREAD

**Issue by zq130320339** [xhzdeng/crpn#25]
论文中的K是什么意思,为什么要用角度除以K

**Comment by Woopython** [comment]
> 论文中的K是什么意思,为什么要用角度除以K

把角度360度量化为K个单位.比如0~15度就对应第1类,15~30度对应第2类...360度一共24类,K就是24.

**Comment by Woopython** [comment]
> 论文中的K是什么意思,为什么要用角度除以K

把角度360度量化为K个单位.比如0到15度就对应第1类,15到30度对应第2类...360度一共24类,K就是24.

**Comment by zq130320339** [comment]
谢谢你的回答,还有一点不理解,pos_map = 1 - pred_map[:, :, 0],这里的pred_map[:,:,0]指的是属于背景的概率吗

**Comment by Woopython** [comment]
> 谢谢你的回答,还有一点不理解,pos_map = 1 - pred_map[:, :, 0],这里的pred_map[:,:,0]指的是属于背景的概率吗

对,pred_map最小单元内的第一个数值表示属于背景的概率,后面的数值表示其方向属于某个类别的概率,所以pred_map[:,:,0]是背景的概率,pos_map表示不是背景的概率.

**Comment by zq130320339** [comment]
好的,请问您有python版本的BalancedSoftmaxWithLoss的代码吗,c++的看起来有点吃力

**Comment by Woopython** [comment]
> 的,请问您有python版本的BalancedSoftmaxWithLoss的代码吗,c++的看起

没有
